FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java

```diff
@@ -820,10 +845,129 @@ private static Mat getEllipseMask(int width, int height) {
                     new Size(width - (2 * paddingX), height - (2 * paddingY)),
                     0
             );
-            Log.i(TAG, (new Size(width - (2 * paddingX), height - (2 * paddingY))).toString());
             ellipseMask = new Mat(height, width, CvType.CV_8UC1, new Scalar(0));
             Imgproc.ellipse(ellipseMask, box, new Scalar(255), -1);
         }
         return ellipseMask;
     }
 }
+
+class Thinning {
+    private boolean B[][];
+
+    Mat doJaniThinning(Mat Image) {
```

Comment:
I'm assuming you copied this code from somewhere. Could you add a comment here pointing at the origin?